rmf_building_map_tools/building_map_model_downloader/building_map_model_downloader.py

```diff
@@ -101,27 +98,16 @@ def download_models(
             else:
                 model_set.add(model.model_name)
 
-    # If we are using ignition fuel tools to download the models
-    # and we do not need to parse them, it means the model directory follows
-    # the ignition gazebo directory structure
-    ign = fuel_tools
-    logging.info("\nUsing Ignition Fuel directory struture : %s\n" % (ign))
     # Ignition fuel tools can only download to this folder, so we set the
     # model path to it
-    if fuel_tools:
-        if export_path is not None:
-            model_path = export_path
-            ign = False
-        else:
-            model_path = "~/.ignition/fuel/"
+    model_path = "~/.ignition/fuel/"
```

Comment:
We should probably start module/function level variables, according to https://google.github.io/styleguide/pyguide.html, we can probably do something like 
```
IGN_FUEL_MODEL_PATH = "~/.ignition/fuel/"
model_path = IGN_FUEL_MODEL_PATH
```

This way, we can probably remove the 2 lines of comments above it too.